internal/python/version.go

```diff
@@ -0,0 +1,49 @@
+package python
+
+import (
+	"log"
+
+	"github.com/Masterminds/semver/v3"
+)
+
+const defaultPython3Version = "3.8"
```

Comment:
3.8 is too old in my opinion. I prefer to pick `3.10` or the latest stable, `3.11`.